dietpi/dietpi-software

```diff
@@ -970,6 +970,13 @@ Available commands:
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/social/#wordpress'
 		aSOFTWARE_DEPS[$software_id]='88 89 webserver'
 		#------------------
+		software_id=206
+		aSOFTWARE_NAME[$software_id]='Homer'
+		aSOFTWARE_DESC[$software_id]='Simple HOMepage for your servER to keep your services on hand'
```

Comment:
homepage and server shouldn't be capitalised here, rest looks good

Reply:
I took this from their GitHub repo, it explains the anagram "HOMER" from "HOMepage" and "servER" 🙂.

Reply:
ohh, didn't see that lol